src/NServiceBus.RabbitMQ/Connection/RabbitMqConnectionFactory.cs

```diff
@@ -44,10 +44,9 @@ public RabbitMqConnectionFactory(ConnectionConfiguration connectionConfiguration
             };
         }
 
-        public virtual IConnection CreateConnection(string purpose)
+        public virtual IConnection CreateConnection(string connectionName)
```

Comment:
I know this is an edge case, but this is potentially breaking if someone is naming the parameter (same applies to the change to `PersistentConnection.ctor()`). Do we have a policy on this? Personally I'm happy to call it non-breaking.